mbuild/lattice.py

```diff
@@ -367,16 +370,71 @@ def populate_xyz(self, x=None, y=None, z=None):
                         tmpx = (val[val_item][0] + i) * a
                         tmp_tuple = tuple((tmpx))
                         cell[key].append(((tmp_tuple)))
+        self.cell = cell
         pp.pprint(cell)
 
+    def write_xyz(self, moleculeName=None, fileName=None):
```

Comment:
Just feed this through something like MDTraj. No point in maintaining our own writers for this.

```python
traj = mb.to_trajectory(my_lattice_compound)
traj.save('foobar.xyz')
```